src/components/PageHeadingFlowRun.vue

```diff
@@ -1,26 +1,77 @@
 <template>
-  <page-heading v-if="flowRun" class="page-heading-flow-run" :crumbs="crumbs">
-    <template #after-crumbs>
+  <div v-if="flowRun" class="page-heading-flow-run">
+    <page-heading class="page-heading-flow-run__heading" :crumbs="crumbs">
+      <template #after-crumbs>
+        <div v-if="flowRun.tags && flowRun.tags.length > 0" class="state-list-item__tags">
+          <p-tag-wrapper v-bind="{ tags: flowRun.tags }" />
+        </div>
+      </template>
+      <template #actions>
+        <template v-if="media.sm">
+          <FlowRunPauseButton :flow-run="flowRun" />
+          <FlowRunResumeButton :flow-run="flowRun" />
+          <FlowRunRetryButton :flow-run="flowRun" />
+          <FlowRunCancelButton :flow-run="flowRun" />
+        </template>
+        <FlowRunMenu :flow-run-id="flowRun.id" :show-all="!media.sm" @delete="emit('delete')" />
+      </template>
+    </page-heading>
+    <div class="page-heading-flow-run__meta">
```

Comment:
I believe this meta can go in the default slot of the `page-heading` component. That would simplify this a bit and remove the need for the root div. 

Maybe you already tried this and it didn't work right?